tests/Frameworks/Laravel/Version_10_x/app/Providers/AppServiceProvider.php

```diff
@@ -19,6 +20,6 @@ public function register(): void
      */
     public function boot(): void
     {
-        //
+        Schema::defaultStringLength(191);
```

Comment:
Just for my understanding and future versions of the framework: What does this have to be set?

Reply:
It's not common in Laravel having to set this odd thing :( but this is the reason https://laravel.com/docs/master/migrations#index-lengths-mysql-mariadb